modules/epomDspBidAdapter.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @name epomDspBidAdapter
+ * @version 1.0.0
+ * @description Adapter for Epom DSP and AdExchange
+ * @module modules/epomDspBidAdapter
+ */
+
+import { logError, logWarn, registerBidder } from '../src/adapters/bidderFactory.js';
+import { config } from '../src/config.js';
+const BIDDER_CODE = 'epomDsp';
+
+export const spec = {
+  code: BIDDER_CODE,
+
+  isBidRequestValid(bid) {
+    const globalSettings = config.getBidderConfig()[BIDDER_CODE]?.epomSettings || {};
+    const hasEndpoint = bid.params?.endpoint || globalSettings.endpoint;
+    return !!(hasEndpoint);
+  },
+
+  buildRequests(bidRequests, bidderRequest) {
```

Comment:
your adapter accepts a bidfloor via it's params, pleas add support for the floors module and `getFloor`

Reply:
added bidFloor method